lib/keys/calendarKeys.ts

```diff
@@ -0,0 +1,196 @@
+import {
+    encryptMessage,
+    getMessage,
+    getSignature,
+    splitMessage,
+    decryptMessage,
+    createMessage,
+    generateKey,
+    VERIFICATION_STATUS,
+    decryptPrivateKey,
+    OpenPGPKey
+} from 'pmcrypto';
+import getRandomValues from 'get-random-values';
+import { c } from 'ttag';
+
+import { ENCRYPTION_TYPES, ENCRYPTION_CONFIGS } from '../constants';
+import { normalize } from '../helpers/string';
+import { serializeUint8Array } from '../helpers/serialization';
+import { hasBit } from '../helpers/bitset';
+import { Address, EncryptionConfig } from '../interfaces';
+import { Key as tsKey, Member } from '../interfaces/calendar';
+
+export const KEY_FLAGS = {
+    PRIMARY: 0b1,
+    ACTIVE: 0b01
+};
```

Comment:
In cases like this, we could write enums, and used in types:

```
export enum KeyFlags {
    PRIMARY = 0b1,
    ACTIVE = 0b01
};
```

Then in `interfaces/calendar/Key.ts`: `Flags: KeyFlags`,
helps us not forget to update interfaces/usages when some flag is removed or changed.